fjord/settings/base.py

```diff
@@ -63,8 +63,8 @@ def path(*dirs):
 USE_L10N = True
 
 # Gettext text domain
-TEXT_DOMAIN = 'messages'
-STANDALONE_DOMAINS = [TEXT_DOMAIN, 'javascript']
+TEXT_DOMAIN = 'django'
+STANDALONE_DOMAINS = [TEXT_DOMAIN, 'djangojs']
```

Comment:
I just pushed a change to this nixing the 'javascript' domain because we don't have any files and doing `./manage.py merge` was kicking up errors.